### PR TITLE
Add DevRel OKRs

### PR DESCRIPTION
**Pull request by campoy**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -120,4 +168,84 @@
...
+
+### Define and Manage a Community / Sales Funnel [DevRel]
+
+- [ ] Refine Salesforce + Pardot MQL and SQL rules
+- [ ] GDPR compliance 
+- [ ] Map and report on conversion funnel with hindsights and recommendations
+
+## Advance MLonCode awareness [DevRel]
+
+### Bring 'Assisted Code Review' to developers in the open-source community [DevRel]
```
Comment (smola) [inline README.md]: Maybe this one should be under the top-level `# Bring 'Assisted Code Review' to developers in the open-source community`?

File: README.md [hunk 2]
```diff
@@ -58,6 +58,54 @@
...
+#### Run beta program on source{d} Engine [DevRel]
+
+- [ ] Identify at least 3 users of source{d} Engine with different personas/use cases
+- [ ] Meet regularly (biweekly) with users and gather feedback
+
+### Defined unified web client [DevRel]
+
+- [ ] Write specifications and draft design for a unified web client (instead of gitbase + bblfsh, etc)
+
+### source{d} Engine add-ons (applications) [DevRel]
```
Comment (smola) [inline README.md]: Maybe these should be under the top-level `# Bring 'Code as Data' to market with enterprises`?

File: README.md (hunk 1 above)
Comment (campoy) [reply README.md]: Done

**Review by smola — APPROVED** [review]

**Comment by campoy** [comment]
ping @eiso

**Pull request merged**